app/controllers/anonymous_feedback/global_export_requests_controller.rb

```diff
@@ -0,0 +1,20 @@
+require "gds_api/support_api"
+
+class AnonymousFeedback::GlobalExportRequestsController < AuthorisationController
+  def create
+    authorize! :request, :global_export_request
+
+    support_api.create_global_export_request(export_request_params)
+    redirect_to anonymous_feedback_explore_path,
+      notice: "We are sending your CSV file to #{current_user.email}. If you don't see it in a few minutes, check your spam folder."
```

Comment:
Is this timeline ("a few minutes") realistic for long reporting periods?  If it's too short it could lead to an increase in zendesk tickets.